spec/rubocop/cop/naming/block_forwarding_spec.rb

```diff
@@ -189,6 +189,15 @@ def foo
           end
         RUBY
       end
+
+      it 'does not register an offense when redefining the block arg' do
```

Comment:
Perhaps `assigning` looks reasonable :-)

```suggestion
      it 'does not register an offense when assigning the block arg' do
```